scripts/installs/start_glassfish_service.bat

```diff
@@ -1 +1,3 @@
-net start "domain1 GlassFish Server"
\ No newline at end of file
+Taskkill /IM domain1Service.exe /F
+ping 192.168.200.201 -n 1 -w 1000 > nul
```

Comment:
Good call, the Start-Sleep is much cleaner and intuitive. I've updated the commit to use that.